src/chrome/content/rules/LitMir.me.xml

```diff
@@ -0,0 +1,15 @@
+<!-- LitMir.me has both a wildcard certificate and a wildcard DNS record. -->
+<ruleset name="LitMir.me">
+	<target host="litmir.me" />
+	<target host="*.litmir.me" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http:" to="https:" />
+
+	<test url="http://www.litmir.me/" />
+
+	<test url="http://ca1w0m0l90013mbinyqq.litmir.me/" />
+	<test url="http://xsu09h57yh41fxtsk0kw.litmir.me/" />
+	<test url="http://667q4b6o4co7fc8f6b84.litmir.me/" />
```

Comment:
They server different content on HTTPS

Reply:
These subdomains are randomly generated and are used to demonstrate the wildcard DNS record.